prediction_market_agent/agents/think_thoroughly_agent/prompts.py

```diff
@@ -1,100 +1,104 @@
-CREATE_OUTCOMES_FROM_SCENARIO_PROMPT = """
-        You are asked about a SCENARIO, which can have multiple, mutually-exclusive outcomes.
-        You should break down the SCENARIO into a list of possible outcomes.
-        
-        Example 1:
-        - SCENARIO: Will Gnosis Pay reach 100,000 users by 2025?
-        - Answer: '''
-            - Gnosis Pay reaches less than 25,000 users by 2025
-            - Gnosis Pay reaches more than 25,000 users but less than 50,000 users by 2025
-            - Gnosis Pay reaches more than 50,000 users but less than 75,000 users by 2025
-            - Gnosis Pay reaches 100,000 users or more by 2025
-        '''
-
-        Example 2:
-        - SCENARIO: Will the price of Bitcoin go up again in the next 2 months?
-        - Answer: '''
-            - The price of crypto will go up in the next 2 months
-            - The price of crypto will not go up in the next 2 months
-        '''
-        
-        [SCENARIO]
-        {scenario}
-        """
-
-CREATE_OUTCOMES_FROM_SCENARIO_OUTPUT = """
-  A list containing multiple bullet points. Each bullet point should start with '-'.
-  Each bullet point should contain a possible outcome resulting from the
-  provided SCENARIO. The produced outcomes should be mutually exclusive, i.e. only one of them should be true whereas
-  the remaining ones should be false.
-  """
+CREATE_REQUIRED_CONDITIONS_PROMPT = """
+You are asked to create a list of conditions that must be met for a SCENARIO to be true.
+
+For example, if the SCENARIO is "Will person X win the election?", the conditional SCENARIOs could be: "Person X must be a candidate in the election", "Person X must not be disqualified from the election", etc.
+
+Write short, concise SCENARIOs, with at most 10 words each, in a form of a question.
+Write up to {n_scenarios} other SCENARIOs that must be met for the SCENARIO to be true.
+
+[SCENARIO]
+{scenario}
+"""
+
+CREATE_HYPOTHETICAL_SCENARIOS_FROM_SCENARIO_PROMPT = """
+You are asked about a SCENARIO, which can have either 'YES' or 'NO' outcomes.
+
+However, based on the SCENARIO, you should create a list of other possible SCENARIOS, these SCENARIOS should also have either 'YES' or 'NO' possible outcomes.
+
+Here are a few examples of SCENARIOs and then SCENARIOs derived from them:
+
+Example 1:
+- SCENARIO: Will Gnosis Pay reach 100,000 users by 2025?
+- Possible other SCENARIOs:
+  - Will Gnosis Pay have less than 25,000 users by 2025?
```

Comment:
Instead of generating outcomes (ie `Gnosis Pay reaches less than 25,000 users by 2025`), I changed it to generate questions like this.

Agents accept the questions as input, so to keep it consistent

Reply:
So you are basically generating additional scenarios than the ones generated before?
Can you give examples on which scenarios were generated before and now, to justify this change?

Reply:
Kind of questions generated before can be seen in the linked issue, for example:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/b395f19d-41a9-4932-b3f6-02d791683f63)

And I screenshotted the kind of questions it generates now into Slack:

![image](https://github.com/gnosis/prediction-market-agent/assets/15619339/2dfb3468-9e5a-470c-a13e-a2801a1f918a)